src/main/java/dev/mcnees/moneymapper/domain/Transaction.java

```diff
@@ -0,0 +1,90 @@
+package dev.mcnees.moneymapper.domain;
+
+import java.time.LocalDate;
+
+public class Transaction {
```

Comment:
Just curious why you used a record elsewhere and not here as well (I haven't done much with records yet so this is likely my ignorance).

Reply:
Records are immutable and don't have setters associated.  I wanted to keep the same object throughout my code and make minor tweaks.
My `Processor`s take a `Transaction` as input, update one or two fields, then output the same `Transaction`.  I could have done this with records but I would need to create a new record for every modification.
IMO my code looked cleaner to me if I kept `Transaction` as a class and used the setters.